### PR TITLE
chore(text): migrate to fast

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: libs/components/src/lib/layout/ui.test.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
 } from '../../visual-tests/visual-tests-utils';
 
 const componentName = 'layout';
-test('should have all connotations', async ({ page }: { page: Page }) => {
+test('should show the component', async ({ page }: { page: Page }) => {
```
Comment (rinaok) [inline libs/components/src/lib/layout/ui.test.ts:10]: Not connected to this branch, it's better to do it here: https://github.com/Vonage/vivid-3/pull/251

File: libs/components/src/lib/text/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# vwc-text
+
+Represents a text custom element.
+The component provisions the Vivid typography font faces and connotations supported by our design system.
+
+##### typography scale
+
+![typography font faces scale image](assets/images/type-ramp.jpeg)
```
Comment (rinaok) [inline libs/components/src/lib/text/README.md:8]: Screen Shot 2022-03-14 at 13 23 12

File: libs/components/src/lib/text/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# vwc-text
+
```
Comment (rinaok) [inline libs/components/src/lib/text/README.md:2]:
```js
<script type='module'>
    import '@vonage/vivid/text';
</script>
```
```suggestion

```

File: libs/components/src/lib/text/README.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+```html preview
+<h1>
+ <vwc-text font-face="caption">
+  lorem ipsum dolor sit amet, consectetur adipiscing elit
+ </vwc-text>
+</h1>
+```
+
+## Properties
```
[the remark this answers is not in the document]
Comment (yinonov) [reply libs/components/src/lib/text/README.md:52]: can u elaborate? or share snippet?

File: libs/components/src/lib/layout/ui.test.ts (hunk 1 above)
Comment (yinonov) [reply libs/components/src/lib/layout/ui.test.ts:10]: it's actually already present in #232. i'll remove

Comment (yinonov) [inline libs/components/src/lib/layout/ui.test.ts:10]:
```suggestion
test('should have all connotations', async ({ page }: { page: Page }) => {
```

Comment (yinonov) [inline libs/components/src/lib/layout/ui.test.ts]:
```suggestion
test('should have all connotations', async ({ page }: { page: Page }) => {
```

File: libs/components/src/lib/text/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (rinaok) [reply libs/components/src/lib/text/README.md:51]: I mean to make an example with each fontface one under another

File: libs/components/src/lib/layout/ui.test.ts (hunk 1 above)
Comment (rinaok) [reply libs/components/src/lib/layout/ui.test.ts]: approved #232

File: libs/components/src/lib/text/README.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+Represents a text custom element.
+The component provisions the Vivid typography font faces and connotations supported by our design system.
+
+##### typography scale
+
+![typography font faces scale image](assets/images/type-ramp.jpeg)
+
+### Basic usage
+
+The following will generate a `headline-1` styled font face
```
Comment (rinaok) [inline libs/components/src/lib/text/README.md:12]: This caption is not relevant anymore

File: libs/components/src/lib/text/text.spec.ts [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+	describe('connotation', () => {
+		it('should set correct internal connotation style', async () => {
+			const connotation = 'info';
+			(element as any).connotation = connotation;
+			await elementUpdated(element);
+
+			const control = element.shadowRoot?.querySelector(`.control.connotation-${connotation}`);
+			expect(control)
+				.toBeInstanceOf(Element);
```
Comment (rinaok) [inline libs/components/src/lib/text/text.spec.ts:31]:
```suggestion
const connotation = 'info';
			expect(getControlElement(element).classList.toString()).toEqual('control');
			(element as any).connotation = connotation;
			await elementUpdated(element);
			expect(getControlElement(element).classList.toString()).toEqual(`control connotation-${connotation}`);
```

Comment (rinaok) [inline libs/components/src/lib/text/text.spec.ts:31]:
```suggestion
const connotation = 'info';
			expect(getControlElement(element).classList.toString()).toEqual('control');
			(element as any).connotation = connotation;
			await elementUpdated(element);
			expect(getControlElement(element).classList.toString()).toEqual(`control connotation-${connotation}`);
```

File: libs/components/src/lib/text/text.spec.ts [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+	});
+
+	describe('tight', () => {
+		it('should set correct internal tight style', async () => {
+			(element as any).tight = true;
+			await elementUpdated(element);
+
+			const control = element.shadowRoot?.querySelector('.control.tight');
+			expect(control)
+				.toBeInstanceOf(Element);
```
Comment (rinaok) [inline libs/components/src/lib/text/text.spec.ts:54]: the same as above

File: libs/components/src/lib/text/text.spec.ts (hunk 6 above)
Comment (yinonov) [reply libs/components/src/lib/text/text.spec.ts:31]: Thanks, wasn't familiar with this helper

**Review by rinaok — APPROVED** [review]

**Pull request merged**